caster/asynch/mouse/legion.py

```diff
@@ -205,6 +205,19 @@ def main(argv):
     monitor = 1
     dimensions = None
     auto_quit = False
+
+    error_code = windll.shcore.SetProcessDpiAwareness(2)  #enable 1-1 pixel mapping
+    if error_code == -2147024891:
+        raise OSError("Failed to set app awareness")
+    dpi_map = {96: 1.0, 120: 1.25, 144: 1.5, 192: 2.0}
+
+    LOGPIXELSX = 88
+    dc = windll.user32.GetDC(0)
+    horizontal_dpi = windll.gdi32.GetDeviceCaps(
+        dc, LOGPIXELSX)  # Horizontal DPI determines the DPI scaling value
+    windll.user32.ReleaseDC(0, dc)
+    dpi_scaling = dpi_map[horizontal_dpi]
```

Comment:
I was just thinking about ways I could test this and that made me wonder if there is any software that could fake it on my current hardware. In order to keep the simulated resolution high enough to be usable it wouldn't necessarily  scale at the same rates horizontally and vertically. It could also have weird DPI values.

What if a piece of hardware doesn't have square pixels? Also, wouldn't it be easier/faster to just divide the given DPI value by 96.0?